### PR TITLE
Javascript Task1

### PR DESCRIPTION
**Pull request by yatellipraveen**
Tasks for day1.

### CODE REVIEW

File: JavaScript/Task_Day1/Basic_Mathematical.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+https://www.codewars.com/kata/57356c55867b9b7a60000bd7/train/javascript
+//Mathematical_Operations
+
+
+function basicOp(operation, value1, value2)
+{ 
+ return eval(value1+operation+value2);
```
Comment (vrallabhandi) [inline JavaScript/Task_Day1/Basic_Mathematical.js]: Avoid using eval. Think of an alternate solution

**Review by vrallabhandi — CHANGES_REQUESTED** [review]
Please make these changes

File: JavaScript/Task_Day1/Transportation_on_Vacation.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+
+// Transportation
+
+
+function rentalCarCost(days) {
+    var total=40*days;
+    if(days>=7)  total=total-50;
+    else if(days>=3) total=total-20;
+    return total;
```
Comment (vrallabhandi) [inline JavaScript/Task_Day1/Transportation_on_Vacation.js]: it is good to have `{ }` for `if-else` blocks as a practice even if they contain a single statement in them.

**Comment by yatellipraveen** [comment]
Hi.. The requested changes are done. Can you please review.

**Pull request closed without merging**